grisera_api/observable_information/observable_information_service_graphdb.py

```diff
@@ -19,10 +18,14 @@ class ObservableInformationServiceGraphDB(ObservableInformationService):
     life_activity_service (LifeActivityService): Service used to communicate with Life Activity
     recording_service (RecordingService): Service used to communicate with Recording
     """
+
     graph_api_service = GraphApiService()
-    modality_service = ModalityServiceGraphDB()
-    life_activity_service = LifeActivityServiceGraphDB()
-    recording_service = RecordingServiceGraphDB()
+
+    def __init__(self):
+        self.modality_service = None
+        self.life_activity_service = None
+        self.recording_service = None
+        self.time_series_service = None
```

Comment:
Missing type definition of these services

Reply:
As above